frontend/src/components/layout/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as Header } from './Header/Header';
+export { default as Footer } from './Footer/Footer';
+export { default as AppNavBar } from './AppNavBar/AppNavBar';
```

Comment:
This is known as a "barrel" file. It allows to 
`import { Header, Footer, ... } from components/layout`
instead of
`import Header from components/layout/Header/Header`
`...`